ui/core.js

```diff
@@ -88,6 +90,10 @@ $.extend( $.ui, {
 		if ( element && element.nodeName.toLowerCase() !== "body" ) {
 			$( element ).blur();
 		}
+	},
+
+	escapeId: function( id ) {
```

Comment:
Use a closure so you can get `idEscape` to be local and then return the actual function. See `.uniqueId()` for an example of this.